src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/web_socket/UnitMoveWebSocketDTO.java

```diff
@@ -0,0 +1,33 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto.web_socket;
+
+import ch.uzh.ifi.hase.soprafs22.rest.dto.PositionDTO;
+
+public class UnitMoveWebSocketDTO {
+    private PositionDTO start;
+    private PositionDTO destination;
+    private Boolean hasMoved;
+
+    public PositionDTO getStart() {
+        return start;
+    }
+
+    public void setStart(PositionDTO start) {
+        this.start = start;
+    }
+
+    public PositionDTO getDestination() {
+        return destination;
+    }
+
+    public void setDestination(PositionDTO destination) {
+        this.destination = destination;
+    }
+
+    public Boolean hasMoved() {
+        return hasMoved;
+    }
+
+    public void setHasMoved(Boolean hasMoved) {
+        this.hasMoved = hasMoved;
+    }
+}
```

Comment:
This file is important for you @apple00juice 
It defines what you'll get from the server when you move a unit on the map.
Are you fine with it or do you need any other properties?

Reply:
I think its good. But Masha updates the socket update, so she might know more... But as she said it works it should be good.